superset-frontend/src/explore/components/DataTablesPane/index.tsx

```diff
@@ -225,7 +225,7 @@ export const DataTablesPane = ({
 
   useEffect(() => {
     if (queriesResponse && chartStatus === 'success') {
-      const { colnames } = queriesResponse[0];
+      const { colnames = [] } = queriesResponse[0];
```

Comment:
codes LGTM, how to reproduce this issue?

Reply:
Reproduce: chart list / welcome -> select birth_names related chart(eg: Pivot Table, Top 10 Boy Name Share).